website/docs/guides/setting-up-dex.md

```diff
@@ -40,7 +41,7 @@ There are a [lot of options][dex-connectors] available with Dex, in this guide w
 use the [GitHub connector][dex-github].
 
 We can get a GitHub ClientID and Client secret by creating a
-[new OAuth appliation][github-oauth].
+[new OAuth application][github-oauth].
```

Comment:
Shouldn't this fix https://staging.docs.gitops.weave.works/typo-fixes/docs/guides/setting-up-dex/#add-credentials?

Reply:
this baffles me

Reply:
I should have looked at https://staging.docs.gitops.weave.works/typo-fixes/docs/next/guides/setting-up-dex/#add-credentials instead - it's fixed there.